src/math/Matrix4.d.ts

```diff
@@ -125,7 +125,7 @@ export class Matrix4 implements Matrix {
 	/**
 	 * Inverts this matrix.
 	 */
-	invert(): Matrix;
+	invert(): Matrix4;
```

Comment:
actually, this line and all other returning this should be like:


```TypeScript
invert(): this
```

Reply:
agreed. we might want to replace them later